src/main/scala/outwatch/util/Store.scala

```diff
@@ -12,7 +12,7 @@ import scala.language.implicitConversions
 
 final case class Store[State, Action](initialState: State,
                                            reducer: (State, Action) => (State, Option[IO[Action]]),
-                                           handler: Observable[Action] with Sink[Action]) {
+                                           handler: Handler[Action, Action]) {
```

Comment:
Why does Store *contain* a `Handler` instead of *being* a Handler?

Reply:
A `Handler` is a `Sink` (`Observable` with `Sink`) and `Sink` is a sealed trait, so nothing can extend `Handler`. It's one of the reasons I initially had the `Handler` wrap the `Observable` and `Sink`.